main.cpp

```diff
@@ -0,0 +1,147 @@
+#include <iostream>
```

Comment:
It is preferable to declare class in .h or .hpp file
define it in other .cpp file and use it in main.cpp